autopark/VehicleType.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace autopark
+{
+    class VehicleType
+    {
+        string typeName;
+        float taxCoefficient; //or float taxCoefficient { get; set; }
+
+        public float TaxCoefficient
+        {
+            get { return taxCoefficient; }
+            set { taxCoefficient = value; }
+        }
+
+        public VehicleType(string typeName, float taxCoefficient)
+        {
+            this.typeName = typeName;
+            this.taxCoefficient = taxCoefficient;
+        }
+
+        public void Display()
+        {
+            Console.WriteLine($"TypeName = {typeName}");
+            Console.WriteLine($"TaxCoefficient = {taxCoefficient}");
+        }
+
+        public override string ToString()
+        {
+            return $"{typeName} {taxCoefficient}";
+        }
+
+        public void ChangeTaxCoef(float n)
+        {
+            taxCoefficient = n;
+        }
```

Comment:
You don't need these method. Use Property instead.

Reply:
![image](https://user-images.githubusercontent.com/79978383/115709628-1b0b6000-a37a-11eb-9222-a47fd1e4eee9.png)
Done